packages/eva/themes/dark.json

```diff
@@ -111,64 +111,64 @@
-  "text-color-info-active": "$color-info-active",
-  "text-color-info-disabled": "$color-info-disabled",
-
-  "text-color-warning": "$color-warning-default",
-  "text-color-warning-active": "$color-warning-active",
-  "text-color-warning-disabled": "$color-warning-disabled",
-
-  "text-color-danger": "$color-danger-default",
-  "text-color-danger-active": "$color-danger-active",
-  "text-color-danger-disabled": "$color-danger-disabled",
+  "background-basic-color-1": "$color-basic-800",
+  "background-basic-color-2": "$color-basic-900",
+  "background-basic-color-3": "$color-basic-1000",
+  "background-basic-color-4": "$color-basic-1100",
+
+  "background-alternative-color-1": "$color-basic-100",
+  "background-alternative-color-2": "$color-basic-200",
+  "background-alternative-color-3": "$color-basic-300",
+  "background-alternative-color-4": "$color-basic-400",
+
+  "background-primary-color-1": "$color-primary-500",
+  "background-primary-color-2": "$color-primary-600",
+  "background-primary-color-3": "$color-primary-700",
+  "background-primary-color-4": "$color-primary-800",
+
+  "border-basic-color-1": "$color-basic-800",
+  "border-basic-color-2": "$color-basic-900",
+  "border-basic-color-3": "$color-basic-1000",
+  "border-basic-color-4": "$color-basic-1100",
+  "border-basic-color-5": "$color-basic-1100",
+
+  "border-alternative-color-1": "$color-basic-100",
+  "border-alternative-color-2": "$color-basic-200",
+  "border-alternative-color-3": "$color-basic-3000",
+  "border-alternative-color-4": "$color-basic-400",
+  "border-alternative-color-5": "$color-basic-500",
+
+  "border-primary-color-1": "$color-primary-500",
+  "border-primary-color-2": "$color-primary-600",
+  "border-primary-color-3": "$color-primary-700",
+  "border-primary-color-4": "$color-primary-800",
+  "border-primary-color-5": "$color-primary-900",
+
+  "text-basic-color": "$color-basic-100",
+  "text-alternate-color": "$color-basic-900",
+  "text-control-color": "$color-basic-100",
+  "text-disabled-color": "$color-basic-700",
+  "text-hint-color": "$color-basic-600",
+
+  "text-primary-color": "$color-primary-default",
+  "text-primary-active-color": "$color-primary-active",
+  "text-primary-disabled-color": "$color-primary-disabled",
+
+  "text-success-color": "$color-success-default",
+  "text-success-active-color": "$color-success-active",
+  "text-success-disabled-color": "$color-success-disabled",
+
+  "text-info-color": "$color-info-default",
+  "text-info-active-color": "$color-info-active",
+  "text-info-disabled-color": "$color-info-disabled",
+
+  "text-warning-color": "$color-warning-default",
+  "text-warning-active-color": "$color-warning-active",
+  "text-warning-disabled-color": "$color-warning-disabled",
+
+  "text-danger-color": "$color-danger-default",
+  "text-danger-active-color": "$color-danger-active",
+  "text-danger-disabled-color": "$color-danger-disabled",
 
   "icon-color-default": "$color-basic-100",
```

Comment:
same for icon [component]-[state/status]-[property]

Reply:
Done